packages/admin/src/layout/FixedLeftRightLayout.tsx

```diff
@@ -1,36 +1,56 @@
+import { makeStyles } from "@material-ui/core";
+import { Theme } from "@material-ui/core/styles";
 import * as React from "react";
 
-const Left: React.SFC<any> = (props) => props.children;
-const Right: React.SFC<any> = (props) => props.children;
+const Left = (props: any): React.ReactElement => props.children;
+const Right = (props: any): React.ReactElement => props.children;
+
 Left.displayName = "FixedLeftRightLayout.Left";
 Right.displayName = "FixedLeftRightLayout.Right";
 
 interface IProps {
     children: any[]; // Array<Left | Right>;
 }
-export class FixedLeftRightLayout extends React.Component<IProps> {
```

Comment:
has this ever been used by anyone??

that is more a POC than anything else

Reply:
If our `grep-all.php` is working correctly, then no, this is not used anywhere except in the stories.  
I will make a separate PR to delete this.

Reply:
@nsams #368